scss/atoms/tables/_tables.scss

```diff
@@ -0,0 +1,95 @@
+/*
+* ==========================================================================
+* Tables
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"mixins";
+
+table {
+	width: 100%;
+	margin: map-get($table-base, margin);
+	font-size: map-get($table-base, font-size);
```

Comment:
consider using em function on this (See https://github.com/jquery/css-chassis/blob/master/scss/atoms/typography/_functions.scss )